samples/http-add-on_with_istio/README.md

```diff
@@ -0,0 +1,189 @@
+$ helm install keda kedacore/keda --namespace keda
+$ helm install http-add-on kedacore/keda-add-ons-http --namespace keda
+```
+
+There should be KEDA now installed and running together with http-add-on
+```
+$ kubectl get po -nkeda
+NAME                                                    READY   STATUS    RESTARTS   AGE
+keda-add-ons-http-controller-manager-79db447c7d-7r9xb   3/3     Running   0          39s
+keda-add-ons-http-external-scaler-dfb9f7bcc-dxqft       2/2     Running   0          39s
+keda-add-ons-http-external-scaler-dfb9f7bcc-fh76m       2/2     Running   0          39s
+keda-add-ons-http-external-scaler-dfb9f7bcc-phcw9       2/2     Running   0          39s
+keda-add-ons-http-interceptor-6dd68bbc87-5jz6v          2/2     Running   0          38s
+keda-add-ons-http-interceptor-6dd68bbc87-5ng2w          2/2     Running   0          39s
+keda-add-ons-http-interceptor-6dd68bbc87-pnp47          2/2     Running   0          38s
+keda-admission-webhooks-554fc8d77f-n7mml                2/2     Running   0          39s
+keda-operator-dd878ddf6-7ww29                           2/2     Running   0          39s
+keda-operator-metrics-apiserver-86cc9c6fff-t4prd        2/2     Running   0          39s
+```
+
+##### Step 3: Deploy sample application
+
+[Podinfo](https://github.com/stefanprodan/podinfo) is a great testing application, we will use it here because it's small, simple to use and
+simple to inspect. Its output ca provide helpful insight into the `Pod` internals.
+```
+$ helm upgrade --install --wait podinfo --namespace default oci://ghcr.io/stefanprodan/charts/podinfo
+```
+
+Let's expose it using istio `VirtualService`.
+```
+$ kubectl apply -f https://raw.githubusercontent.com/kedify/examples/main/samples/http-add-on_with_istio/manifests/istio.yaml
+```
+
+Your application should be now available on the istio gateway IP address under host www.podinfo.com.
+```
+$ GATEWAY_IP=$(kubectl get svc -nistio-system istio-ingressgateway -o json | jq --raw-output '.status.loadBalancer.ingress[0].ip')
+$ curl -H "host: www.podinfo.com" "http://$GATEWAY_IP"
+```
+```json
+{
+  "hostname": "podinfo-5965fc9856-4tfpg",
+  "version": "6.6.3",
+  "revision": "b0c487c6b217bed8e6a53fca25f6ee1a7dd573e3",
+  "color": "#34577c",
+  "logo": "https://raw.githubusercontent.com/stefanprodan/podinfo/gh-pages/cuddle_clap.gif",
+  "message": "greetings from podinfo v6.6.3",
+  "goos": "linux",
+  "goarch": "amd64",
+  "runtime": "go1.22.3",
+  "num_goroutine": "8",
+  "num_cpu": "16"
+}
+```
+
+The `podinfo` is running in `default` `Namespace` with the istio sidecar injected and operational.
+```
+$ kubectl get po -ndefault
+NAME                       READY   STATUS    RESTARTS   AGE
+podinfo-5965fc9856-4tfpg   2/2     Running   0          22s
+```
+
+##### Step 4: Start autoscaling
+
+First we will need to create `HTTPScaledObject` to tell KEDA and http-add-on what metrics to use and which `Deployment` to scale
```

Comment:
Once we have the build with `kedify-http` scaler, let's start using ScaledObject in favor of HTTPScaledObject

Reply:
I noticed the examples in this repo all use upstream KEDA and are without Kedify agent, so I followed that pattern too. 

I will make sure to list Kedify agent as a prerequisite and reference this:
https://kedify.io/resources/blog/kedify-keda-powered-public-beta-launch-announcement